### ISSUE TITLE
Can we remove this winreg package, as there is no such package for mac or linux

### ISSUE THREAD

**Issue by yeahydq** [ZhuJD-China/RainbowGPT#5]
> https://github.com/ZhuJD-China/RainbowGPT/blob/1d79068258cecbe61c3523c1533170c336050dae/Rainbow_utils/get_google_result.py#L9](url)`

**Comment by ZhuJD-China** [comment]
Of course, if you only need to modify the local VPN proxy port in China without directly reading the Windows system proxy from the registry, you can do so.
